vars/jobs.groovy

```diff
@@ -14,7 +14,7 @@ def get_job_url(job){
 def build_job(name, parameters, is_stable)
 {
   def job = null
-  job = build job: name, parameters: parameters, propagate: false
+  job = build job: name, parameters: parameters, propagate: false, wait: true
```

Comment:
with this wait we will not be able to run jobs in parallel am I right?

Reply:
No, it has nothing common with parallel job execution.